### PR TITLE
feat: glew support for latest OpenGL

### PR DESCRIPTION
**Pull request by petrSchreiber**

### CODE REVIEW

File: src/glfw-sandbox/engine.hpp [hunk 1]
```diff
@@ -90,6 +49,47 @@ class WindowCreator
...
+
+		// Needed before GLEW
+		glfwMakeContextCurrent(window);
+
+		// Get the latest GL, yeaaah!
+		int err = glewInit();
+		if (GLEW_OK != err)
+		{
+			std::cout << "Failed to initialize GLEW library: " << glewGetErrorString(err) << std::endl;
+		}
```
Comment (petrSchreiber) [inline src/glfw-sandbox/engine.hpp:87]: Latest OpenGL loader

File: src/glfw-sandbox/engine.hpp [hunk 2]
```diff
@@ -90,6 +49,47 @@ class WindowCreator
...
+
+		// Get the latest GL, yeaaah!
+		int err = glewInit();
+		if (GLEW_OK != err)
+		{
+			std::cout << "Failed to initialize GLEW library: " << glewGetErrorString(err) << std::endl;
+		}		
+
+		// Initialize OpenGL defaults
+		opengl_intialize();
```
Comment (petrSchreiber) [inline src/glfw-sandbox/engine.hpp:90]: Let's do it by default

**Review by milansky — COMMENTED** [review]
$(SolutionDir)\thirdparty\glew-2.2.0\lib\Release\x64\glew32s.lib seems to be missing
Not sure if I need to compile it on my PC and include it...

File: src/glfw-sandbox/engine.hpp (hunk 1 above)
Comment (milansky) [reply src/glfw-sandbox/engine.hpp:87]: Very nice and clean :)

**Comment by milansky** [comment]
Update: not working until GLEW libs are added :( 
Once added to thirdparty\glew folder all loads nicely.

**Comment by petrSchreiber** [comment]
Hi @milansky,

excellent catch, I pushed the two libs to the commit. Thank you!

**Comment by milansky** [comment]
You may merge :)

**Pull request merged**